boxes/templates/boxes/box_list.html

```diff
@@ -113,9 +114,9 @@ <h2 class="text-lightest title">MANAGE BOXES</h2>
     <button type="button" class="md-close btn-none"><i class="fa fa-times fa-lg" aria-hidden="true"></i></button>
     <div class="row center-xs center-md center-lg">
       <div class="col-xs-12 col-sm-12 col-md-12">
-        <h3 class="subtitle center-xs text-lightest">CREATE NEW BOX</h3>
-        <p class="smalltext text-lightest text-mob-"> For others to be able to send you encrypted content through Hawkpost, you'll need to generate "boxes".</p>
-        <p class="smalltext text-lightest">  Each box has an associated link that you can copy and share with any given person. If you want, you can set the expiration date and number of messages after which the link will be locked and no longer accessible.</p>
+        <h3 class="subtitle center-xs text-lightest">{% trans "CREATE NEW BOX" %}</h3>
+        <p class="smalltext text-lightest text-mob-">{% trans "For others to be able to send you encrypted content through Hawkpost, you'll need to generate "boxes"." %}</p>
```

Comment:
`"boxes"` causes an error, since this it closes the string and is interpreted as a variable. Please use `\"boxes\"`.